Dockerfile

```diff
@@ -21,24 +21,48 @@ RUN git clone ${MAPSERVER_REPO} --branch=${MAPSERVER_BRANCH} --depth=100 /src
 COPY checkout_release /tmp
 RUN cd /src; /tmp/checkout_release ${MAPSERVER_BRANCH}
 
+COPY instantclient /tmp/instantclient
+
+ARG WITH_ORACLE=OFF
+
+RUN (if test "${WITH_ORACLE}" = "ON"; then \
+       apt-get update && \
+       LC_ALL=C DEBIAN_FRONTEND=noninteractive apt-get install -y libarchive-tools libaio-dev && \
+       apt-get clean && \
+       rm -rf /var/lib/apt/lists/* && \
+       mkdir -p /opt/instantclient && \
+       cd /opt/instantclient && \
+       (for i in /tmp/instantclient/*.zip; do bsdtar --strip-components=1 -xvf $i; done) && \
+       ls -ltr /opt/instantclient && \
+       ln -s libnnz19.so /opt/instantclient/libnnz18.so; \
+     else \
+       mkdir -p /opt/instantclient/sdk && \
+       touch /opt/instantclient/dummy.txt; \
```

Comment:
Why do we need this file?

Reply:
also  the "ln -s" statement is surprising :) (version 19 -> 18)

Reply:
I suspected that the "COPY --from=builder /opt/instantclient /usr/local/lib/" might fail, but it does not. I've just forced pushed without this line

Reply:
> also the "ln -s" statement is surprising :) (version 19 -> 18)

this is to workaround an issue in MapServer detection of Oracle SDK. Adressed upstream per https://github.com/MapServer/MapServer/pull/6146

Reply:
ok makes sense